boot.php

```diff
@@ -17,7 +17,7 @@
  * easily as email does today.
  */
 
-require_once('include/autoloader.php');
+require_once(__DIR__ . DIRECTORY_SEPARATOR. 'vendor' . DIRECTORY_SEPARATOR . 'autoload.php');
```

Comment:
Yes, but @Quix0r is doing it in his massive PR, and I don't want to trigger more conflicts than what this will already do.